cdap-relational-spi/src/main/java/io/cdap/cdap/common/internal/relational/DefaultSqlDialectConverterProvider.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common.internal.relational;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.name.Named;
+import io.cdap.cdap.common.conf.CConfiguration;
+import io.cdap.cdap.common.conf.Constants;
+import io.cdap.cdap.etl.spi.relational.SqlDialectConverter;
+
+import javax.annotation.Nullable;
+
+public class DefaultSqlDialectConverterProvider implements Provider<SqlDialectConverter> {
```

Comment:
This entire package can be moved out of the SPI package to a different package where the calcite SPI implementation will be used.